esp/applications/hpcc_services/hpcc_services.yaml

```diff
@@ -0,0 +1,178 @@
+hpcc_services:
+   WsSMC:
+      ActivityInfoCacheSeconds: 10
+      ActivityInfoCacheAutoRebuildSeconds: 120
+   WsWorkunits:
+      ViewTimeout: 1000
+      AWUsCacheMinutes: 15
+      AllowNewRoxieOnDemandQuery: false
+      LayoutProgram: dot/dot -Tsvg -Gordering=out
+      EclServer: ''
+      EclServerQueue: ''
+      StyleSheets:
+        xslt:
+        - name: atts
+          ^: "/esp/xslt/atts.xslt"
+        - name: dot_update
+          ^: "/esp/xslt/dot_update.xslt"
+        - name: dot
+          ^: "/esp/xslt/dot.xslt"
+        - name: graphStats
+          ^: "/esp/xslt/graphStats.xslt"
+        - name: graphupdate_gvc
+          ^: "/esp/xslt/graphupdate_gvc.xslt"
+   WsTopology:
+      warnIfCpuLoadOver: 95
+      warnIfFreeMemoryUnder: 5
+      warnIfFreeStorageUnder: 5
+      AllowNewRoxieOnDemandQuery: false
+      SystemUseRewrite: false
+   WsDfu:
+      DisableUppercaseTranslation: false
+      ViewTimeout: 1000
+      EclServer: ''
+      EclServerQueue: ''
+      StyleSheets:
+        xslt:
+        - name: def_file
+          ^: "./smc_xslt/def_file.xslt"
+   WsDfuXRef:
+      ViewTimeout: 1000
+      LayoutProgram: dot/dot -Tsvg -Gordering=out
+      StyleSheets:
+        xslt:
+        - name: def_file
+          ^: "./smc_xslt/def_file.xslt"
+   FileSpray:
+      QueueLabel: dfuserver_queue
+      MonitorQueueLabel: dfuserver_monitor_queue
+      RootFolder: /c$/thordata
+   ws_elk:
+      ELKIntegration:
+         Kibana: ''
+         ElasticSearch: ''
+         LogStash: ''
+         ELKIntegration: ''
+   ws_store:
+      StoreProvider:
+         lib: dalistorelib
+      Stores:
+         Store:
+         -  name: HPCCApps
+            description: Generic KeyVal store for HPCC Applications
+            default: true
+   ws_machine:
+      excludePartitions: /dev*,/sys*,/proc*,/run*,/boot
+      monitorDaliFileServer: false
+      UseDefaultSSHUserID: false
+      UseDefaultPIDFileName: true
+      ProcessFilters:
+        Platform:
+        - name: Windows
```

Comment:
Windows? 

What does this section mean anyway?

Reply:
Purely rebuilding the internal filters for "preflight".  I'm hoping this all goes away or moves for both windows and linux.  Perhaps the preflight pages can be or are removed for containerized?  I imagine we will just use standard cloud tools for monitoring in the future.